microsoft-edge/webview2/how-to/add-host-object-to-script.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Use app objects in JavaScript
+description: Use host object in JavsScript using the WebView2 AddHostObjectToScript API.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 2/17/2022
+---
+# Use host objects in JavaScript
+
+WebView2 enables applications to bridge between web and native sides of an application using an object that can be passed between the two sides. Such objects are defined in the native code and often called host objects. They can be projected into JavaScript using a WebView2 API called `AddHostObjectToScript`, as described in this document.
+
+Why use `AddHostObjectToScript`?
+
+  * Developers often encounter a native object whose methods or properties they want to call from JavaScript. They may want these native object methods to be triggered by web code or by user interaction on the web side.
```

Comment:
Global in this article: try to use 2nd-person tense.  https://docs.microsoft.com/en-us/style-guide/grammar/person#in-general-use-second-person